pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -828,19 +828,16 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *v1.Serv
 
 	status.Ingress = []v1.LoadBalancerIngress{{IP: loadbalancer.VipAddress}}
 
-	port, err := getPortByIP(lbaas.network, loadbalancer.VipAddress)
-	if err != nil {
-		return nil, fmt.Errorf("Error getting port for LB vip %s: %v", loadbalancer.VipAddress, err)
-	}
-	floatIP, err := getFloatingIPByPortID(lbaas.network, port.ID)
+	portID := loadbalancer.VipPortID
```

Comment:
It's included in an integration test](https://github.com/openstack/neutron-lbaas/blob/f99f4f8a6dd5893703bbc66fdbb29d26d73fc23b/neutron_lbaas/tests/unit/db/loadbalancer/test_db_loadbalancerv2.py#L767) so I'm pretty sure `vip_port_id` is returned in the create response. We've also been running this in a dev env for over a week, and things are fixed. 

@coreypobrien are we doing create ops?